webui/react/src/components/ExperimentActions.tsx

```diff
@@ -0,0 +1,169 @@
+import { Button, Popconfirm } from 'antd';
+import React, { useCallback, useState } from 'react';
+
+import { archiveExperiment, killExperiment, launchTensorboard, setExperimentState,
+} from 'services/api';
+import { ExperimentDetails, RunState, TBSourceType } from 'types';
+import { openCommand } from 'utils/routes';
+import { cancellableRunStates, killableRunStates, terminalRunStates } from 'utils/types';
+
+import css from './ExperimentActions.module.scss';
+
+interface Props {
+  experiment: ExperimentDetails;
+  finally: () => void; // A callback to trigger after an action is done.
```

Comment:
**nit:** on callbacks I've been using the format of `on[Action]` so for this maybe we can name it `onComplete`.

Reply:
sg

Reply:
with `finally` I'm trying to borrow from the async api pattern and to communicate that this will get called wether the action successfully completes or it fails.

Reply:
`onSetteled` ? after https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled

Reply:
`onSettled` sounds good to me!